vdbench_wrapper/ci_test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -x
```

Comment:
Raul taught me a bash trick to prevent command failures from being ignored:
```
set -xeEo pipefail
```
Highly recommend.   This avoids a lot of cases where benchmark fails but CI reports success or fails much later.

Reply:
Changed it.